### PR TITLE
i18n(es): Updated `environment-variables.mdx` & `netlify.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `environment-variables.mdx` & `netlify.mdx` based on these commits 9f6a54c2efaa77d17df25890a2a58f85427d2cef, c9aed6ab6abfdcd8bb804743a03cee485007b46a & ac6c9e67056660dbffaa1238d94bcd4f125aee35

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/netlify.mdx [hunk 1]
```diff
@@ -164,6 +164,28 @@ Una vez que ejecutes `astro build`, se generará un archivo `dist/_redirects`. N
 Todavía puedes incluir un archivo `public/_redirects` para redirecciones manuales. Cualquier redirección que especifiques en la configuración de redirecciones se añadirá al final de las redirecciones que hayas definido tú mismo.
 :::
 
+### Constructores bajo demanda
+
+[Los constructores bajo demanda de Netlify](https://docs.netlify.com/configure-builds/on-demand-builders/) son funciones serverless usadas para generar contenido web según sea necesario que se almacena automáticamente en el CDN de Netlify. Puedes habilitar estas funciones usando la [configuración de `builders`](#builders).
+
+Por defecto, todas las páginas se renderizarán en la primera visita y el resultado renderizado se reutilizará para cada visita posterior hasta que vuelvas a desplegar. Para establecer un tiempo de revalidación, llama a la [`runtime.setBuildersTtl(ttl)` local](/es/guides/middleware/#locals) con la duración (en segundos).
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/netlify.mdx:171]:
```suggestion
Por defecto, todas las páginas se renderizarán en la primera visita y el resultado renderizado se reutilizará para cada visita posterior hasta que vuelvas a desplegar. Para establecer un tiempo de revalidación, llama al [`runtime.setBuildersTtl(ttl)` local](/es/guides/middleware/#locals) con la duración (en segundos).
```

File: src/content/docs/es/guides/integrations-guide/netlify.mdx [hunk 2]
```diff
@@ -164,6 +164,28 @@ Una vez que ejecutes `astro build`, se generará un archivo `dist/_redirects`. N
...
+Como un ejemplo, para el siguiente fragmento, Netlify almacenará el HTML renderizado durante 45 segundos.
+
+```astro
+---
+import Layout from '../components/Layout.astro';
+if (import.meta.env.PROD) {
+  Astro.locals.runtime.setBuildersTtl(45);
+}
+---
+<Layout title="Astro on Netlify">
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/netlify.mdx:182]:
```suggestion
<Layout title="Astro en Netlify">
```

File: src/content/docs/es/guides/integrations-guide/netlify.mdx [hunk 3]
```diff
@@ -164,6 +164,28 @@ Una vez que ejecutes `astro build`, se generará un archivo `dist/_redirects`. N
...
+  {new Date(Date.now())}
+</Layout>
+```
+
+Es importante tener en cuenta que los constructores bajo demanda ignoaran los parámetros de consulta al comprobar las páginas en caché. Por ejemplo, si `example.com/?x=y` está en caché, se servirá para `example.com/?a=b` (parámetros de consulta diferentes) y `example.com/` (sin parámetros de consulta) también.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/netlify.mdx:187]:
```suggestion
Es importante tener en cuenta que los constructores bajo demanda ignoran los parámetros de consulta al comprobar las páginas en caché. Por ejemplo, si `example.com/?x=y` está en caché, se servirá para `example.com/?a=b` (parámetros de consulta diferentes) y `example.com/` (sin parámetros de consulta).
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por el review @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**